db/migrate/20210504084958_create_qualification_results.rb

```diff
@@ -0,0 +1,11 @@
+class CreateQualificationResults < ActiveRecord::Migration[6.1]
+  def change
+    create_table :qualification_results, id: :uuid do |t|
+      t.belongs_to :qualification, null: false, foreign_key: true, type: :uuid
+      t.string :subject, null: false
```

Comment:
Do we want to `default: ""`? We spoke about it before (I can't remember where 😂 ) and I don't think we reached a consensus, but we've been doing it ever since

Reply:
Hmmm 🤔  I don't _think_ so, given that having one of these records doesn't make sense without both of these fields being present.

Reply:
Do non-secondary qualifications also have QualificationResults? If so, they need to sometimes have grades without subjects.

Reply:
For simplicity, they do not - non-secondary qualifications keep the subject and the grade on the `Qualification` record